public/js/post/post_components.js

```diff
@@ -139,9 +147,11 @@ var SpecialistPost = React.createClass({
                   secondYearCourses={secondYearCourses} 
                   laterYearCourses={laterYearCourses} 
                   textBoxNumbers={[3, 3, 4]} 
+                  disabledTextboxes={[true, true, false]}
```

Comment:
@christinem I don't like having two parallel lists that must be the same length. Probably pass in a nested list instead.